azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/redux/middleware/handler/CallingMiddlewareActionHandler.kt

```diff
@@ -291,8 +291,7 @@ internal class CallingMiddlewareActionHandlerImpl(
         coroutineScope.launch {
             callingService.getParticipantsInfoModelSharedFlow().collect {
                 if (isActive) {
-                    val participantUpdateAction = ParticipantAction.ListUpdated()
-                    participantUpdateAction.participantMap = it
+                    val participantUpdateAction = ParticipantAction.ListUpdated(HashMap(it))
```

Comment:
There was a bug here, where "it" is installed into the store from calling SDK, it's re-using a map.

Need to copy the map in order for state to be immutable (when tracking user added/removed, this map was always up to date, i.e. old and new state the same).

Reply:
👍